include/RAJA/config.hpp.in

```diff
@@ -178,6 +187,33 @@ const int DATA_ALIGN = @RAJA_DATA_ALIGN@;
 #define RAJA_RESTRICT __restrict__
 #endif
 
+//
+// Runtime bounds checking for Views
+//
+#if defined(ENABLE_BOUNDS_CHECK)
+#define BOUNDS_CHECK
```

Comment:
This should probably be changed to RAJA_BOUNDS_CHECK_INTERNAL (or something similarly unique) so we don't conflict with other project's macro definitions.